components/codeflare/resources/codeflare-rules.tmpl.yaml

```diff
@@ -0,0 +1,145 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: codeflare-rules
+  namespace: [[ .Monitoring.Namespace ]]
```

Comment:
```suggestion
  namespace: {{ .Monitoring.Namespace }}
```

Reply:
Regarding this, it was creating issues because the rule files has other entries of `{{  }}` so I had to change the delimiter to `[[]]`